neptune/internal/mockpoolutils.go

```diff
@@ -204,6 +204,10 @@ func MakeHierarchyVertex(vertexLabel, code, codeLabel string, numberOfChildren i
 	return vertex
 }
 
+func SetOrder(vertex *graphson.Vertex, order int) {
+	setVertexTypedProperty("g:Int64", vertex, "order", map[string]interface{}{"@type": "g:Int64", "@value": float64(order)})
```

Comment:
Is there a reason it can't just be a `float64` instead of an `int`? It would remove the need to cast it.

Reply:
Yes, there is no reason. I just followed other properties, I can change both to use float64